samples/sqltransport/startup-purge-behavior/SqlTransport_6/Receiver/TruncateTableStartupTask.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.Data.SqlClient;
+using NServiceBus;
+
+#region TruncateTableAtStartup
+
+public class TruncateTableAtStartup : INeedInitialization
+{
+    public void Customize(EndpointConfiguration configuration)
+    {
+        var connectionString = @"Data Source=.\SqlExpress;Database=SQLServerTruncate;Integrated Security=True;Max Pool Size=100";
```

Comment:
Is it worth showing how to get the connection string from the `EndpointConfiguration`, if possible?

Reply:
I don't know that I know how to do that at this stage of the pipeline, we'd have to actually connect to the database in case someone has provided a connection factory and not a connection string. Any suggestions?

Reply:
I think we already do that in the transport somewhere by doing something like:

```
using(var c = connectionFactory())
{
   var connectionString = c.ConnectionString;
}
```

I'm not sure, though, we can do that at this stage. @tmasternak?

Reply:
I can't figure out how to do it in a way that doesn't involve reading 2 magic string settings from the settings bag. Doing that makes the sample look more confusing that it needs to be.